driver-core/src/main/com/mongodb/internal/operation/ChangeStreamBatchCursorHelper.java

```diff
@@ -26,20 +26,25 @@
 import java.util.Collections;
 import java.util.List;
 
+import static com.mongodb.internal.operation.ServerVersionHelper.FOUR_DOT_FOUR_WIRE_VERSION;
 import static java.util.Arrays.asList;
 
 final class ChangeStreamBatchCursorHelper {
-    private static final List<Integer> UNRETRYABLE_SERVER_ERROR_CODES = asList(136, 237, 280, 11601);
+    private static final List<Integer> RETRYABLE_SERVER_ERROR_CODES =
+            asList(6, 7, 63, 89, 91, 133, 150, 189, 234, 262, 9001, 10107, 11600, 11602, 13388, 13435, 13436);
     private static final List<String> NONRESUMABLE_CHANGE_STREAM_ERROR_LABELS = asList("NonResumableChangeStreamError");
```

Comment:
The label is still valid when connected to older servers. Also, the label has been retained in 4.4 (see SERVER-46103) to avoid backward breaking changes.

Reply:
Per offline conversation in Slack, the server is still generating this label, but that's to ensure that new server versions will work with old driver versions that still expect it. This PR should completely remove the blacklist label and only use the whitelist label or the manually-maintained list of error codes to determine if a change stream error is resumable.

Reply:
As discussed with Divjot in Slack, the check for NONRESUMABLE_CHANGE_STREAM_ERROR_LABELS is redundant to the error code check, and so can be removed.  Good looking out, Ross!

Reply:
I have removed the error label.